common/src/main/scala/com/gu/recipeasy/models/CuratedRecipe.scala

```diff
@@ -69,6 +69,15 @@ case class TimesInMins(
   cooking: Option[Double]
 )
 
+object TimesInMinsAdapted {
+  def preparationTimeInMinutesFromCuratedRecipe(cr: CuratedRecipe): Double = {
```

Comment:
You don't need the `FromCuratedRecipe` part. This is observed from the function parameter.